crates/lib/src/qpu/quilc/mod.rs

```diff
@@ -212,16 +212,17 @@ MEASURE 1 ro[1]
 
     #[tokio::test]
     async fn run_compiled_bell_state_on_qvm() {
-        let config = Configuration::load().await.unwrap_or_default();
+        let config = ClientConfiguration::load().await.unwrap_or_default();
+        let client = QcsClient::with_config(config);
```

Comment:
This will require `#[derive(Default)]` on `QcsClient`

```suggestion
        let client = QcsClient::load().await.unwrap_or_default();
```